backend/src/models/db-questions.js

```diff
@@ -0,0 +1,82 @@
+const sqlite3 = require('sqlite3')
+const db = new sqlite3.Database('./db/questions.db')
+
+function get_all_questions(data) {
+  // Lookup all questions
+  return new Promise((resolve, reject) => { 
+    db.all('SELECT rowid, uid, title, body FROM questions', (err, rows) => {
+    
+      // Check for a qeury error
+      if (err) {
+        reject(data)
+      }
+
+      // Transform the requested data into the appropriate format
+      data = rows.map(({rowid, uid, title, body}) => ({
+        id: rowid,
+        meta: {uid},
+        title,
+        body
+      }))
+      resolve(data)
```

Comment:
this is where the data is resolved. Data object passed in is not necessary. (in fact it's overwritten by the assignment on line 15).

Reply:
Fixed